### DISCUSSION TITLE
From scalar integrals to known scaled variables.

### DISCUSSION THREAD

**Discussion by sunnyasquabla** [FeynCalc/feyncalc#257] [Q&A]
Hi,
Greetings,

can someone tell me how can I reduce the scalar integrals into known scaled variables?
For example,

![image](https://github.com/FeynCalc/feyncalc/assets/107980747/900da3ef-73df-4900-8d8a-ba3c011fd338)

Reference: https://arxiv.org/abs/1712.05234

Thank You

**Comment by vsht** [comment]
Hi,
the limit can be probably taken with Package-X. If not, then I don't see a simple way to rederive that automatically.
